data-prepper-api/src/test/java/org/opensearch/dataprepper/model/configuration/SinkModelTest.java

```diff
@@ -148,8 +149,8 @@ void sinkModel_with_include_keys() {
         final Map<String, Object> pluginSettings = new LinkedHashMap<>();
         final SinkModel sinkModel = new SinkModel("customSinkPlugin", Arrays.asList("routeA", "routeB"), null, Arrays.asList("bcd", "abc", "efg"), null, pluginSettings);
 
-        assertThat(sinkModel.getExcludeKeys(), equalTo(new ArrayList<String>()));
-        assertThat(sinkModel.getIncludeKeys(), equalTo(Arrays.asList("bcd", "abc", "efg")));
+        assertThat(sinkModel.getExcludeKeys(), is(empty()));
+        assertThat(sinkModel.getIncludeKeys(), contains("abc", "bcd", "efg")); //must be sorted
```

Comment:
@Mishail , This is not verifying the exact order. If you need them sorted, then you should assert equality.

```
assertThat(sinkModel.getIncludeKeys(), equalTo(List.of("abc", "bcd", "efg"));
```

Reply:
@Mishail , Thank you for clarifying that. The name itself is unclear, but this should work.